templates/semantic-ui/zenodo_rdm/macros/record_item.html

```diff
@@ -71,7 +85,9 @@
       {# Communities list #}
       <h5>
         {% set communities_entries = record.parent.communities.entries %}
-        <b>{{ _("Part of:")}} {{ communities_list(communities=communities_entries) }}</b>
+        {% if communities_entries %}
+          <b>{{ _("Part of:")}} {{ communities_list(communities=communities_entries) }}</b>
```

Comment:
nit: should we [truncate](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.truncate) this? Or maybe we can show something like "Part of: <comm1>, <comm2> and 3 more". 

Ignore it if it's not relevant to the agreed solution.

Reply:
Discussed this with Lars and decided to display all the communities here